org.eclipse.draw2d.tests/META-INF/MANIFEST.MF

```diff
@@ -2,14 +2,17 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Plugin.name
 Bundle-SymbolicName: org.eclipse.draw2d.tests
-Bundle-Version: 3.13.100.qualifier
+Bundle-Version: 3.14.0.qualifier
+Import-Package: org.slf4j
 Bundle-Vendor: %Plugin.providerName
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.core.resources;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.draw2d;bundle-version="[3.16.0,4.0.0)",
+ org.eclipse.draw2d.examples;bundle-version="3.15.400",
  org.junit,
- org.eclipse.jface;bundle-version="[3.2.0,4.0.0)"
+ org.eclipse.jface;bundle-version="[3.2.0,4.0.0)",
+ org.eclipse.swtbot.swt.finder;bundle-version="4.2.1"

```

Comment:
I guess greater or equal to 4.2.1 would be better here or?

Reply:
Isn't that exactly what's described here?

![image](https://github.com/user-attachments/assets/27f45369-afe9-42c6-8fe1-1541de3a5310)

Reply:
I always thought for ranges you need some range qualifier. Just checked on the OSGI spec and learned I was wrong. Really need to deepen my OSGI knowledge. An OSGI book is already on my reading list. Recommendations for more are warmly welcome.